app/views/concerns/paginated_json_api_result_set_view.rb

```diff
@@ -59,4 +59,8 @@ def pagination_total_pages
       end
     end
   end
+
+  def pagination_page_item_count
```

Comment:
Right... yes, let's do that, i.e. keep `paginated_set.count` in `PaginatedView#pagination_page_item_count` and override in `Portal::Index` with `paginated_set.docs.count`.